src/App.js

```diff
@@ -8,7 +16,20 @@ class App extends Component {
   }
 
   render() {
-    return (<Calculator />);
+    return (
+      <>
+        <Router>
+          <NavBar />
+          <section className="content">
+            <Routes>
+              <Route path="/math-magicians" element={<Home />} />
```

Comment:
- [ ] Please add the correct path for the home page which is `/` because now nothing is rendered when home path is opened see the image below

![image](https://user-images.githubusercontent.com/68796232/179875857-b6fb42b5-156b-4c95-9957-295b2db9a291.png)